lnwallet/interface.go

```diff
@@ -660,3 +662,25 @@ func SupportedWallets() []string {
 
 	return supportedWallets
 }
+
+// DefaultSendOpOptions is the default set of options for SendOutputs function.
+func DefaultSendOpOptions() *SendOpOptions {
```

Comment:
nit: what does "Op" stand for? Should we just call it `SendOutputOption` everywhere?